From: Simon McVittie <smcv@debian.org>
Date: Tue, 24 Dec 2019 12:07:01 +0000
Subject: tests: Install test.portal for installed-tests

Otherwise, when running installed-tests, xdg-desktop-portal will
find a real installed backend such as xdg-desktop-portals-gtk (if any),
rather than using the intended test backend.

Fixes: https://github.com/flatpak/xdg-desktop-portal/issues/431
Signed-off-by: Simon McVittie <smcv@debian.org>
---
 tests/Makefile.am.inc | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/tests/Makefile.am.inc b/tests/Makefile.am.inc
index 7054ed7..f14dbe3 100644
--- a/tests/Makefile.am.inc
+++ b/tests/Makefile.am.inc
@@ -101,9 +101,11 @@ if ENABLE_INSTALLED_TESTS
 endif
 
 installed_test_dbsdir = $(installed_testdir)/dbs
+installed_test_portalsdir = $(installed_testdir)/portals
 
 if ENABLE_INSTALLED_TESTS
 dist_installed_test_dbs_DATA = tests/dbs/no_tables
+dist_installed_test_portals_DATA = tests/portals/test.portal
 endif
 
 dist_test_scripts = \
@@ -115,10 +117,6 @@ test_programs += \
 	test-permission-store \
 	$(NULL)
 
-EXTRA_DIST += \
-        tests/portals/test.portal \
-        $(BNULL)
-
 DISTCLEANFILES += \
 	tests/services/org.freedesktop.portal.Documents.service \
 	tests/services/org.freedesktop.impl.portal.PermissionStore.service \
